core/node/vm_runner/Cargo.toml

```diff
@@ -16,13 +16,16 @@ zksync_dal.workspace = true
 zksync_contracts.workspace = true
 zksync_state.workspace = true
 zksync_storage.workspace = true
+zksync_core.workspace = true
 vm_utils.workspace = true
 
 tokio = { workspace = true, features = ["time"] }
 anyhow.workspace = true
 async-trait.workspace = true
 once_cell.workspace = true
 tracing.workspace = true
+futures = { workspace = true, features = ["compat"] }
```

Comment:
Nit: Using `futures` just for `BoxFuture` is redundant; it's just a type alias which you can define yourself (also see the comment about using Tokio's `JoinHandle`s).

Reply:
Yep, no need for `BoxFuture` anymore anyway